src/human_aware_rl/ppo/run_ppo_bc_experiments.sh

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env bash
+# This file contains the script to generate the ppo self-play agents for the 5 classic layouts
+
+#First we need to create the BC-agents, the agent will be saved in the bc_runs directory under human_aware_rl/bc_runs directory
+python "$(dirname "$0")/../imitation/reproduce_bc.py"
```

Comment:
Maybe specify which directory this bash script should be run from as a comment in the file too?

Reply:
Got it. It actually doesn't really matter where the bash script is run since this path is relative to where this script is, not where it is being called. I'll add some comments clarifying that.

Reply:
Ah makes sense – my bad for not realizing that